DependencyInjection/Configuration.php

```diff
@@ -102,18 +102,31 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->arrayNode('sitemap')
```

Comment:
@dbu didn't we want to change/extend this? You wanted to use single configurations too, right?